packages/wd-base/src/_typography.scss

```diff
@@ -0,0 +1,123 @@
+/* ==========================================================================
+   Typography
+   ========================================================================== */
+
+/* Font Faces
+   ========================================================================== */
+$font-primary: "Circular", sans-serif !default;
+$font-secondary: "Open Sans", sans-serif !default;
```

Comment:
Idk if vitor is still using Open Sans.

Reply:
I'll remove the secondary font, since it seems like we are only using Circular